certbot_route53/authenticator.py

```diff
@@ -14,112 +16,128 @@
 
 logger = logging.getLogger(__name__)
 
-TTL = 30
+INSTRUCTIONS = (
+    "To use, create an IAM user and attach the AmazonRoute53FullAccess policy, then store "
```

Comment:
I'll have to admit being a newbie when it comes to IAM. If you create a role and attach it to an EC2 instance, does boto3 automatically get the credentials from somewhere?

For `AmazonRoute53FullAccess`, when trying to set up an IAM role to be able to use Route53, I only see "Managed Policies," of which the closest match is `AmazonRoute53FullAccess`. I notice there is a more specific policy in sample-aws-policy.json, though. How does one go about applying that policy to a role?

Ideally we could figure out something that's both best-practices-friendly and newbie-accessible enough to document as part of the plugin.